include/nuttx/kthread.h

```diff
@@ -166,6 +166,8 @@ int kthread_create(FAR const char *name, int priority, int stack_size,
  *
  ****************************************************************************/
 
+int nxtask_delete(pid_t pid);
```

Comment:
it's better to move nxtask_delete and nxtask_create to include/nuttx/sched.h
and change the inclusion at line 30 to nuttx/sched.h